operator/pkg/apis/istio/v1alpha1/validation/validation.go

```diff
@@ -59,14 +67,10 @@ func firstCharsToUpper(s string) string {
 	return strings.Join(res, ".")
 }
 
-func deprecatedSettingsMessage(values *valuesv1alpha1.Values) string {
+func deprecatedSettingsMessage(values *valuesv1alpha1.Values, iops *v1alpha1.IstioOperatorSpec) string {
+	const mixerDeprecatedMessage = "telemetry v2 configs enabled by default"
```

Comment:
instead of mentioning telemetry v2, can we instead say something like "Mixer is deprecated. It will be removed from Istio with the 1.8 release. Please consult our docs on the replacement." ?

I'd like to focus on what is deprecated primarily. And those using Mixer for policy likely won't see the connection to telemetry v2.

Reply:
ok done